### PR TITLE
Add blobserve rules for new vscode insiders

### PR DESCRIPTION
**Pull request by jeanp413**
## Description

Adds new blobserve rules required by new version of vscode insiders and next stable version to be released this week hopefully. 
Related https://github.com/gitpod-io/gitpod/pull/7930

## Related Issue(s)

Fixes #

## How to test

Test already done in https://github.com/gitpod-io/gitpod/pull/7930 but just in case
- Open a workspace using vscode stable, then open simple browser extension webview should load successfully

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **princerachit** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @princerachit` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- **[components/blobserve/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/blobserve/OWNERS)**
- **[installer/pkg/components/blobserve/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/pkg/components/blobserve/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by kylos101** [comment]
@jeanp413 without this change, how would that impact users? I ask because if it will impact users (positive or negative) we should consider adding a release note.

**Comment by kylos101** [comment]
Works well:
image

**Comment by kylos101** [comment]
Version
```
Version: 1.63.2
Commit: 16d43d2bb89bb32b58ff0c84bef55486269760d6
Date: 2022-01-12T05:43:47.372Z
Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.99 Safari/537.36
```

**Comment by kylos101** [comment]
@jeanp413 may I ask you to open a small PR for [this file too](https://github.com/gitpod-io/ops/blob/main/deploy/workspace/manifests/blobserve.yaml)? This is needed to ensure when we deploy workspace-clusters, that they are patched with the proper values.

**Comment by jeanp413** [comment]
> @jeanp413 without this change, how would that impact users? I ask because if it will impact users (positive or negative) we should consider adding a release note.

This change should have no user impact as the rules are ignored for the current vscode stable and insiders versions

> @jeanp413 may I ask you to open a small PR for this file too? This is needed to ensure when we deploy workspace-clusters, that they are patched with the proper values.

Good catch @kylos101, thanks for pointing that out

File: components/blobserve/pkg/blobserve/blobspace.go
Comment (kylos101) [inline components/blobserve/pkg/blobserve/blobspace.go:219]: Nit: just curious, why the change from xerrors? I ask because of [this section on error handling](https://www.notion.so/gitpod/How-we-develop-e8ce7e562478458a9223eb9b797415b2#2c483985b5c244fc8546287f9d0fba36).

**Comment by jeanp413** [comment]
@kylos101 PR created https://github.com/gitpod-io/ops/pull/982

**Review by kylos101 — COMMENTED** [review]
@jeanp413 just a quick question on error handling. :smile:

File: components/blobserve/pkg/blobserve/blobspace.go
Comment (jeanp413) [reply components/blobserve/pkg/blobserve/blobspace.go:219]: Before it was creating a new error with `xerrors.Errorf` and returning that, what we want here is to just log if an error occurred while trying to apply the replace rule but continue the execution of the remaining rules as some of them are expected to fail because some files could not exist (vscode renamed some files but this code should work for stable version too containing old filenames)

**Review by kylos101 — APPROVED** [review]
/LGTM

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 65edd4da4b9eaf85b63a91e020a6f11cae79a81b</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7990#pullrequestreview-871082716" title="LGTM">kylos101</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [kylos101]
- ~~[components/blobserve/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/blobserve/OWNERS)~~ [kylos101]
- ~~[installer/pkg/components/blobserve/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/pkg/components/blobserve/OWNERS)~~ [kylos101]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7990#issuecomment-1028339883" title="Approved">kylos101</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/7990#issuecomment-1028339883" title="Approved">kylos101</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [kylos101]
- ~~[components/blobserve/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/blobserve/OWNERS)~~ [kylos101]
- ~~[installer/pkg/components/blobserve/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/pkg/components/blobserve/OWNERS)~~ [kylos101]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
